.github/sync/RELEASE.md

```diff
@@ -364,9 +378,12 @@ Currently, there are only 2 activities we use rever for, (1) aggregating the aut
 
 </details>
 
-## 8. Open PRs to bump [Anaconda Recipes][Anaconda Recipes] and [conda-forge][conda-forge] feedstocks to use `YY.M.0`.
+## 9. Open PRs to bump [Anaconda Recipes][Anaconda Recipes] and [conda-forge][conda-forge] feedstocks to use `YY.M.0`.
+
+> **Note**
+> Conda-forge's PRs will be auto-created via the `regro-cf-autotick-bot`. If any changes need to be made to the recipe that were not automatically added, create a PR to push additional changes to the auto-generated PR.
```

Comment:
There is no way to add more commits to autotick's branch (even via a PR)

Instead branch from autotick's branch add any additional commits and open a new PR